### PR TITLE
Small B3 updates to report template

### PR DESCRIPTION
**Pull request by biyeun**
up for grabs cc @emord

### CODE REVIEW

File: corehq/apps/reports/templates/reports/bootstrap3/base_template.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {% block js %}{{ block.super }}
     {% compress js %}
     <script src="{% new_static 'hqwebapp/js/ajax_csrf_setup.js' %}"></script>
-    <script src="{% new_static 'style/js/bootstrap2/main.js' %}"></script>
+    <script src="{% new_static 'style/js/bootstrap3/main.js' %}"></script>
```
Comment (czue) [inline corehq/apps/reports/templates/reports/bootstrap3/base_template.html]: i don't know what else is implied by this change, but confirming it's not something that we think needs any QA?

**Comment by czue** [comment]
fyi, I think @emord is offline so might make sense to be be pinging someone else

File: corehq/apps/reports/templates/reports/bootstrap3/base_template.html (hunk 1 above)
Comment (biyeun) [inline corehq/apps/reports/templates/reports/bootstrap3/base_template.html]: Confirmed. The files are basically identical

**Comment by biyeun** [comment]
@czue does this look ok to merge?

**Pull request merged**